packages/buttons/src/elements/Anchor.tsx

```diff
@@ -10,22 +10,15 @@ import PropTypes from 'prop-types';
 import { StyledAnchor, StyledExternalIcon } from '../styled';
 
 export interface IAnchorProps extends AnchorHTMLAttributes<HTMLAnchorElement> {
-  /** Apply danger styling */
+  /** Determines if danger styling is used */
   isDanger?: boolean;
   /**
-   * Used when the anchor navigates to an external resource. Applies `target="_blank"`
-   * along with `rel="noopener noreferrer"` to ensure safe
-   * [cross-origin destination links](https://web.dev/external-anchors-use-rel-noopener/).
+   * Determines if the anchor navigates to an external resource. When enabled, target="_blank"
+   * and rel="noopener noreferrer" attributes are added to ensure safe [cross-origin destination links](https://web.dev/external-anchors-use-rel-noopener/).
    **/
```

Comment:
Please reduce line length by adding a newline at about the 80-100 column position (in this case, after the word "safe" to keep the markup link valid). We don't want developers to have to parse long lines of comment text.

Additional [nit] All JSDoc comments start with `/**` and, preferably, end with `**/` (rather than `*/`).

Reply:
Fixed in latest push.